ios/Podfile.lock

```diff
@@ -916,9 +916,9 @@ SPEC CHECKSUMS:
   Onfido: e4a69434fc8ea8369d01ce77e5c2bdfa675dc40e
   onfido-react-native-sdk: f68229759cc52aee2a577f866852ffae8430ec83
   OpenSSL-Universal: 1aa4f6a6ee7256b83db99ec1ccdaa80d10f9af9b
-  Permission-LocationAccuracy: e8adff9ede1b23b43b7054a4500113d515fc87a8
-  Permission-LocationAlways: 7f7f373d086af7a81b2f4f20d65d29266ca2043b
-  Permission-LocationWhenInUse: 3ae82a9feb5da4e94e386dba17c7dd3531af9feb
+  Permission-LocationAccuracy: 76669f87b4c276f5ae803cc0ddd1862a4c0e9dd8
+  Permission-LocationAlways: a274bc04bb386068782468dbdaca3859f51634ca
+  Permission-LocationWhenInUse: 3a2b0dbc167d79e8e920a4377ff9520cdc108407
```

Comment:
All of these changes need to be removed.

Reply:
Ok

Reply:
?

Reply:
Should be Resolved
![Screenshot from 2022-06-14 01-10-18](https://user-images.githubusercontent.com/51829206/173462430-6c699b2a-b7ed-463d-8a0d-01b266f2bcb3.png)